oracle/types/market.go

```diff
@@ -86,14 +82,21 @@ func NewProviderMarketMap(name string, tickerConfigs TickerToProviderConfig) (Pr
 			return ProviderMarketMap{}, fmt.Errorf("expected provider config name %s, got %s", name, config.Name)
 		}
 
-		offChainMap[config.OffChainTicker] = ticker
+		if offChainMap[config.OffChainTicker] == nil {
+			offChainMap[config.OffChainTicker] = []mmtypes.Ticker{ticker}
+			continue
+		}
+
+		offChainMap[config.OffChainTicker] = append(offChainMap[config.OffChainTicker], ticker)
```

Comment:
This doesn't right, the mapping should be 1:1 with the tickerConfigs

Reply:
The wonky behavior of duplicate feeds (this goes away with our proposed fix